docs/kratos/session-management/25_revoke-sessions-hook.md

```diff
@@ -0,0 +1,21 @@
+---
+id: revoke-sessions-hook
```

Comment:
@vinckr could you quickly add this to the sidebar?

Reply:
not needed in this case as its using the whole folder autogenerated.